src/app/app.module.ts

```diff
@@ -2,13 +2,15 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
+import { GreetingComponent } from './greeting/greeting-message.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    GreetingComponent
   ],
   imports: [
-    BrowserModule
+    BrowserModule,  AppComponent
```

Comment:
no need to add AppComponent into imports since it is a local component to AppModule